cmd/ebrelayer/txs/relay.go

```diff
@@ -40,32 +42,38 @@ func RelayEvent(chainId string, cdc *amino.Codec, passphrase string, claim *type
 
 	err := cliCtx.EnsureAccountExistsFromAddr(claim.Validator)
 	if err != nil {
-		fmt.Errorf("Validator account error: %s", err)
+		fmt.Printf("Validator account error: %s", err)
 	}
 
 	msg := ethbridge.NewMsgMakeEthBridgeClaim(*claim)
 
 	err1 := msg.ValidateBasic()
 	if err1 != nil {
-		fmt.Errorf("Msg validation error: %s", err1)
+		fmt.Printf("Msg validation error: %s", err1)
 	}
 
 	cliCtx.PrintResponse = true
 
 	//prepare tx
 	txBldr, err = utils.PrepareTxBuilder(txBldr, cliCtx)
 	if err != nil {
+		fmt.Printf("Msg prepare error: %s", err)
 		return err
 	}
 
 	// build and sign the transaction
-	txBytes, err := txBldr.BuildAndSign("validator", passphrase, []sdk.Msg{msg})
+	txBytes, err := txBldr.BuildAndSign(validatorName, passphrase, []sdk.Msg{msg})
```

Comment:
Ahh, "Validator" is updated to a parameterized validatorName here. This answers my question.